StabilityClient.Net/StabilityClient.Net/StabilityClient.Net.csproj

```diff
@@ -1,9 +1,34 @@
 <Project Sdk="Microsoft.NET.Sdk">
-
     <PropertyGroup>
         <TargetFramework>net6.0</TargetFramework>
         <ImplicitUsings>enable</ImplicitUsings>
         <Nullable>enable</Nullable>
     </PropertyGroup>
-
-</Project>
+    <ItemGroup>
+        <Protobuf Include="Protos/api.proto" GrpcServices="client" ProtoRoot="Protos" />
+        <Protobuf Include="Protos\dashboard.proto" GrpcServices="Client">
+            <SourceUrl>https://raw.githubusercontent.com/Stability-AI/api-interfaces/main/src/proto/dashboard.proto</SourceUrl>
+        </Protobuf>
+        <Protobuf Include="Protos\engines.proto" GrpcServices="Client">
+            <SourceUrl>https://raw.githubusercontent.com/Stability-AI/api-interfaces/main/src/proto/engines.proto</SourceUrl>
+        </Protobuf>
+        <Protobuf Include="Protos\generation.proto" GrpcServices="Client" ProtoRoot="Protos">
+            <SourceUrl>https://raw.githubusercontent.com/Stability-AI/api-interfaces/main/src/proto/generation.proto</SourceUrl>
+        </Protobuf>
+        <Protobuf Include="Protos\project.proto" GrpcServices="Client">
+            <SourceUrl>https://raw.githubusercontent.com/Stability-AI/api-interfaces/main/src/proto/project.proto</SourceUrl>
+        </Protobuf>
+        <Protobuf Include="Protos\tensors.proto" GrpcServices="Client">
+            <SourceUrl>https://raw.githubusercontent.com/coreweave/tensorizer/fdaed5d2ece40a1aeccf8b396092ce801f5a3910/proto/tensors.proto</SourceUrl>
+        </Protobuf>
+    </ItemGroup>
+    <ItemGroup>
+        <PackageReference Include="Google.Protobuf" Version="3.21.9" />
+        <PackageReference Include="Grpc.Net.Client" Version="2.49.0" />
+        <PackageReference Include="Grpc.Net.ClientFactory" Version="2.32.0" />
+        <PackageReference Include="Grpc.Tools" Version="2.49.0">
+            <PrivateAssets>all</PrivateAssets>
+            <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+        </PackageReference>
+    </ItemGroup>
+</Project>
```

Comment:
missing endline

Reply:
Dodałam tę linię, poszedł push, ale nie wiem dlaczego nie widać jej tutaj jako zmiana w pliku

Reply:
![image](https://user-images.githubusercontent.com/91951071/202180872-06bfaef3-7ee7-4615-abc5-295a397d7d97.png)
Github nie zaznacza, że brakuje linii to znaczy, że on tam jest :)